code/game/machinery/suit_storage_unit.dm

```diff
@@ -301,7 +302,6 @@
 
 	if(panel_open)
 		wires.Interact(user)
-		return
```

Comment:
Without this return, when you use a multitool/wirecutter on the unit to open the wires, you will also hit the unit with your tool.